qutebrowser/config/configtypes.py

```diff
@@ -1375,14 +1399,6 @@ def validate(self, value):
         elif 'always' in values and len(values) > 1:
             raise configexc.ValidationError(
                 value, "List cannot contain always!")
-        # Values have to be valid
-        elif not set(values).issubset(set(self.valid_values.values)):
-            raise configexc.ValidationError(
-                value, "List contains invalid values!")
-        # List can't have duplicates
-        elif len(set(values)) != len(values):
-            raise configexc.ValidationError(
-                value, "List contains duplicate values!")
 
     def complete(self):
```

Comment:
I think something like this should be added to `FlagList` as well so there's completion for all combinations as well. Maybe `FlagList` also should have `combinable_values`, but set to `None` by default - then `complete` uses that if set (for `ConfirmQuit` or `valid_values` otherwise (for `UrlSegmentList`)